.github/workflows/docker-publish.yaml

```diff
@@ -1,11 +1,27 @@
-name: Publish Docker
+name: Deploy
 on:
-  push:
-    branches:
-      - main
-      - 1.52.0.maintenance
-    paths:
-      - 'docker/fusionauth/release-trigger'
+  workflow_dispatch:
+    inputs:
+      command:
+        type: choice
+        options:
+          - build    # build only
+          - publish  # build & publish to dockerhub
+        default: build
+        description: Build only or Build And Publish to DockerHub
+      version:
```

Comment:
I get the desire but are there any examples where we'd actually want to override our Savant version? That would be kind of precedent breaking. We do this in preview ECR builds, but that's different.